src/node.cc

```diff
@@ -866,7 +866,10 @@ static ExitCode InitializeNodeWithArgsInternal(
 
     for (const auto& file_path : file_paths) {
       std::string path = cwd + kPathSeparator + file_path;
-      per_process::dotenv_file.ParsePath(path);
+      auto file_is_exist = per_process::dotenv_file.ParsePath(path);
+
+      if (!file_is_exist)
+        errors->push_back(std::string(file_path) + ": not found");
```

Comment:
Why do we need `std::string(..)` on a `std::string`?

Reply:
i missed this, will fix this soon on the next commit